.github/workflows/build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Setup Java
         uses: actions/setup-java@v4
         with:
-          distribution: zulu
+          distribution: oracle
```

Comment:
This change is certainly ok but it's only the compile time dependency. AEM will still be running the aem-sdk images we create ourselves which are based on zulu-openjdk